package/MDAnalysis/analysis/align.py

```diff
@@ -832,9 +888,9 @@ def __init__(self, mobile, reference=None, select='all', filename=None,
         self.ref_frame = ref_frame
 
         self.filename = filename
-        self.universe = mda.Merge(self.mobile_atoms)
+        self.results.universe = mda.Merge(self.mobile_atoms)
```

Comment:
FYI @joaomcteixeira @PicoCentauri : mdacli will need to be able to deal with whole Universes in the results. This can be as simple as reporting that this result cannot be meaningfully saved. (Although, you can serialize a Universe nowadays — just the question if you want to).